### PR TITLE
Add Docker containers for rapids

### PR DESCRIPTION
**Pull request by lu-wang-dl**
Manually upudate the docker image to https://hub.docker.com/layers/196534986/starboy19850515/rapids-databricks/latest/images/sha256-2bc566038fd1dbc3ab029dd2003d25e69851647757ae7edab7f34efebc40e5d2?context=repo.

And tested with a Databricks cluster:
Screen Shot 2022-03-18 at 12 41 05 PM
Screen Shot 2022-03-18 at 12 48 25 PM

Screen Shot 2022-03-18 at 12 38 43 PM

### CODE REVIEW

File: ubuntu/gpu/cuda-11/rapids/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM databricksruntime/gpu-conda:cuda11
+
+COPY rapids-spec.txt /tmp/spec.txt
+
+RUN conda create --name rapids --file /tmp/spec.txt && \
+    rm -f /tmp/spec.txt
+
+# Set an environment variable used by Databricks to decide which conda environment to activate by default.
+ENV DEFAULT_DATABRICKS_ROOT_CONDA_ENV=rapids
```
Comment (panchalhp-db) [inline ubuntu/gpu/cuda-11/rapids/Dockerfile:9]: Does this work with newer runtimes without conda?

**Review by panchalhp-db — APPROVED** [review]
Were you able to get this image to download and run a cluster? Curious to know what the difference was compared to the last time we tried it.

File: ubuntu/gpu/cuda-11/rapids/Dockerfile (hunk 1 above)
Comment (lu-wang-dl) [reply ubuntu/gpu/cuda-11/rapids/Dockerfile:9]: Tried  to start the DCS cluster with DBR 9.1 and 10.4, both works. Added into the description.

**Comment by lu-wang-dl** [comment]
@panchalhp-db Not doing anything special, it just works. Also interested to know what happens.  Could you help check whether there is any difference between your  Dockerfile and the one in this PR?

**Comment by mohammadsubhani** [comment]
@lu-wang-dl, is this stable on 10.4 LTS ST cluster? if is showing waiting to run commands for nearly 10 mins. 
 
Screenshot 2022-03-21 at 7 07 22 PM

Screenshot 2022-03-21 at 7 08 15 PM

**Pull request merged**